### PR TITLE
Feature/cvsb 9040

### PR DESCRIPTION
**Pull request by bsiddappa**
Integration tests fixed. Working locally as well on jenkins.

### CODE REVIEW

**Comment by KHodderDVSA** [comment]
Ignore the 7949 commit message - just forgot to rename somehow. It's 9040 content

File: package.json [hunk 1]
```diff
@@ -12,77 +12,56 @@
...
-    "serverless": "^1.45.0",
+    "lambda-tester": "^3.6.0",
+    "serverless": "^1.56.1",
     "serverless-dynamodb-local": "^0.2.37",
-    "serverless-offline": "^3.33.0",
+    "serverless-offline": "^5.12.0",
     "serverless-plugin-tracing": "^2.0.0",
     "serverless-plugin-typescript": "^1.1.5",
-    "sinon": "^7.4.1",
+    "sinon": "^7.5.0",
```
Comment (arupdvsa) [inline package.json]: this can be removed

Comment (KHodderDVSA) [reply package.json]: No it can't. Bhavana didn't get rid of ALL instances

Comment (KHodderDVSA) [reply package.json]: And I don't have time to do it now

Comment (arupdvsa) [reply package.json]: okay in that case can we create a ticket so we may keep track of this and can re-visit later

Comment (arupdvsa) [reply package.json]: created tech debt ticket https://jira.dvsacloud.uk/browse/CVSB-9496 to capture revisiting Sinon

Comment (KHodderDVSA) [reply package.json]: created https://jira.dvsacloud.uk/browse/CVSB-9496 to do this

**Pull request merged**